orangecontrib/explain/widgets/tests/test_owpermutationimportance.py

```diff
@@ -324,20 +324,20 @@ def test_sparse_data(self):
         sparse_model = RandomForestLearner(random_state=0)(sparse_data)
         self.send_signal(self.widget.Inputs.data, sparse_data)
         self.send_signal(self.widget.Inputs.model, sparse_model)
-        self.wait_until_finished()
+        self.wait_until_finished(10000)
         self.assertFalse(self.widget.Error.domain_transform_err.is_shown())
         self.assertFalse(self.widget.Error.unknown_err.is_shown())
 
         model = RandomForestLearner(random_state=0)(data)
         self.send_signal(self.widget.Inputs.data, sparse_data)
         self.send_signal(self.widget.Inputs.model, model)
-        self.wait_until_finished()
+        self.wait_until_finished(10000)
```

Comment:
I think it should be 
```suggestion
        self.wait_until_finished(timeout=10000)
```